tests/test_rest_api/test_action/test_automatic_review.py

```diff
@@ -169,48 +143,30 @@ class TestRunReviewer(TestBase):
 
     def setUp(self):
         self.init_request = Mock(form={'run_id': 'run1'})
-        with patch.object(AutomaticLaneReviewer, 'eve_get', return_value=[passing_lane]):
+        with patch.object(AutomaticReviewer, 'eve_get', return_value=failing_lanes):
             self.reviewer1 = ar.AutomaticRunReviewer(self.init_request)
 
-    def test_eve_get(self):
```

Comment:
@tcezard How come this test was removed too? Aren't we still using Eve?

Reply:
Agreed, I'd reinstate this. The tests for this are a bit messy so don't know if we want to do this now, but I'd put `test_eve_get` and others into a base test class, since `eve_get` is part of `AutomaticReviewer`